### PR TITLE
image-meta-tag

### PR DESCRIPTION
**Pull request by marqh**
plausible replacement for #209 (with updates from master)

### CODE REVIEW

**Comment by malcolmbrooks** [comment]
The last update to #209 had the tags move from 0.4.1 to 0.4.2 by the way.

Malcolm

File: image-meta-tag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    name: image-meta-tag
+    version: "0.4.2"
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: 0.4.2
+
+build:
+    number: 0
+    skip: True
```
Comment (bjlittle) [inline image-meta-tag/meta.yaml]: Really? ... is that what you want to do here? No build then :cry:

**Comment by malcolmbrooks** [comment]
In #209 I had:
   skip: True  # [win and osx]

but as #209 wasn't testing properly I wasn't sure if that was correct

**Comment by bjlittle** [comment]
Thanks @malcolmbrooks and @marqh :+1:

**Pull request merged**